packet.go

```diff
@@ -52,6 +59,10 @@ func Parse(b, secret []byte) (*Packet, error) {
 		return nil, errors.New("radius: invalid packet length")
 	}
 
+	if len(cryptoAuthenticator) != 0 && len(cryptoAuthenticator) != 16 {
```

Comment:
I am not sure if I can find the redundant statement. I wanted to allow an empty cryptoAuthenticator or one with length 16.

* empty is acceptable when there are no encrypted fields.
* 16 is the only allowed value otherwise.

Reply:
Sorry misread the line... statement is okay.